synapse/tests/test_lib_certdir.py

```diff
@@ -50,7 +50,7 @@ def test_certdir_host(self):
     def test_certdir_hostca(self):
         with self.getCertDir() as cdir:
             cdir.genCaCert('syntest')
-            cdir.genHostCert('visi.vertex.link', signas='syntest')
+            cdir.genHostCert('visi.vertex.link', signas='syntest', sans='DNS:visi.vertex.link')
```

Comment:
do we need to set the sans string here? wouldn't  it automatically be set to visi.vertex.link? earlier comment about validating the content applies.